app/src/engine/uci_engine.cpp

```diff
@@ -329,8 +329,7 @@ std::vector<std::string> UciEngine::lastInfo() const {
 
     if (last_info.empty()) {
         Logger::warn<true>(
-            "Warning; No info line found in the last line which includes the "
-            "score from {}",
+            "Warning; Last info string with score (cp) not found from {}",
```

Comment:
why `(cp)` .. this is wrong `score mate` or `score cp`

Reply:
right, updated

Reply:
okay but why did you know add eval? there is no eval in the last info string line.. we expect score not eval

Reply:
oh, i thought theyre the same thing(?) but removed it anyways